sbws/lib/relaylist.py

```diff
@@ -68,6 +82,40 @@ def measured(self):
         # return [r for r in relays if r.measured is not None]
         return [r for r in relays if not r.is_unmeasured]
 
+    def relay_ed25519_master_key(self, ns):
```

Comment:
I think this function doesn't belong in RelayList (it does something for a single relay, not something related to the RelayList as a whole); if the relay class is implemented as I envision, this function will aide the creation of `Relay.__init__` and then disappear.

Reply:
i agree, i put it there cause is where currently the info for bw lines is coming from.